docs/APIRef.ActionsOnElement.md

```diff
@@ -116,7 +123,7 @@ await element(by.id('scrollView')).scrollTo('top');
 
 Simulates typing of the specified text into the element, using the system's builtin keyboard and typing behavior.
 
-On iOS, any element can be typed into, as long as it can become first responder and conforms to the [`UITextInput`](https://developer.apple.com/documentation/uikit/uitextinput) protocol.
+On iOS, any element can be typed into, as long as it can become first responder and conforms to the [`UITextInput`](https://developer.apple.com/documentation/uikit/uitextinput) protocol. Before typing the text, Detox attempts making the element the first responder, if it isn't already. If the element’s window is not key window, Detox attempts making it the key window.
```

Comment:
"...Detox attempts [at] making the element the first responder..."?

Reply:
No, it attempts making the element the first responder.

That's correctly worded.

Reply:
source?

Reply:
The English language.